### DISCUSSION TITLE
It is upload all content only to rename folders?

### DISCUSSION THREAD

**Discussion by mrexojo** [abraunegg/onedrive#2694] [Q&A] [answered]
After renaming folders in my laptop, onedrive sync command is uploaing  as new content for the new name folder, instead of rename the original to new name and don't touch the files.

It's normal? I'd need to know to rename from onedrive website and don't wait to upload my content everytime that I rename folders.

thanks

**Comment by abraunegg** [comment, marked as answer]
@mrexojo 

If you rename your local folders, the client *should* (based on all tested when developing the client) rename the folders online automatically when using `--monitor`. This is not possible when using standalone mode `--synchronize` as the actual local rename event is not known by the client. Please read: https://github.com/abraunegg/onedrive/blob/master/docs/known-issues.md#moving-files-into-different-folders-should-not-cause-data-to-delete-and-be-re-uploaded

If you rename the folder online, so long as the folder is already in-sync locally, on next sync, the local folder will be renamed.
